_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+
+![Głowica termostatyczna](/assets/img/posts/2022-01-11-smarthome/windowsensor.png){: width="250" }
+
+#### Głowice termostatyczne
+
+Temat głowic termostatycznych może być nieco bardziej zawiły, bo dostępnych jest sporo wariantów. 
+Ja wybrałem [Moes BRT-100-TRV](https://www.zigbee2mqtt.io/devices/BRT-100-TRV.html). Przy wyborze kierowałem się liczbą wystawianych informacji oraz zapewnieniem producenta, 
+że działają na baterii dłużej niż inne warianty.
+Szczegółowe informacje o wystawianych danych dostępne są na stronach projektu [Zigbee2MQTT](https://www.zigbee2mqtt.io/supported-devices/).
+Jedna z moich głowic raportuje stan baterii 91%, druga 83%. Obydwie działają od listopada, a w momencie sprawdzania tego stanu mieliśmy połowę stycznia.
+W kwestii baterii mogę dodać tylko, że ze względu na różnice w napięciu między bateriami a akumulatorami (1.2V vs 1.5V) konieczne może być używanie zwykłych baterii.
+
+![Głowica termostatyczna](/assets/img/posts/2022-01-11-smarthome/trv.png){: width="250" }
+
+#### Czujniki wycieku wody
+
+Konstrukcyjnie dosyć proste urządzenia. Na ich spodzie znajdują się dwie elektrody, których zwarcie powoduje zmianę stanu czujnika.
+W przypadku czujników Xiaomi/Aqara elektrody możemy wykręcić, zmniejszając tym samym poziom, od którego reagują.
+Najstarszy z moich czujników ma już dwa lata, a ciągle raportuje stan baterii na 100%, więc bez obaw można umieszczać je w trudno dostępnych miejscach.
+
+![Czujnik wycieku wody](/assets/img/posts/2022-01-11-smarthome/leaksensor.png){: width="250" }
+
+#### Czujniki temperatury
+
+Jedne z moich pierwszych czujników, a jak wspominałem, zaczynałem z bramką od Xiaomi, stąd wybór też tego producenta.
+Z zalet kwadratowych czujników temperatury Xiaomi mogę wymienić to, że poza temperaturą oraz wilgotnością raportują też ciśnienie atmosferyczne.
+
+![Czujnik temperatury](/assets/img/posts/2022-01-11-smarthome/temperaturesensor.png){: width="250" }
+
+## Bezpieczeństwo i prywatność
+
+W mojej ocenie w całej zabawie w SmartHome warto zwrócić uwagę na prywatność oraz bezpieczeństwo.
+Nie wyobrażam sobie trzymać w sieci urządzeń, które zbierają dane i wysyłają je na Azjatyckie, niesprawdzone serwery.
+W sieci można znaleźć wiele przykładów kamerek, które albo miały słabe zabezpieczenia, albo luki w bezpieczeństwie i pozwalały każdemu na zdalny dostęp.
+Dla mnie podstawowym wymaganiem przy tego typu rozwiązaniach jest praca w sieci lokalnej.
+Wszystkie smart urządzenia mają wycięty dostęp do internetu, a pozwalam na niego jedynie przy okresowych aktualizacjach.
+W przyszłości zastanawiam się nad poluźnieniem tej restrykcji dla wybranych serwerów i urządzeń, ponieważ koliduje mi to z integracją od pralki LG oraz zmywarki Bosch.
+Aktualnie, jeśli mam potrzebę dostępu z zewnątrz to zapewniam sobie ten dostęp sam za pomocą rozwiązań, które uważam za godne zaufania. 
```

Comment:
Aktualnie, jeśli mam potrzebę dostępu z zewnątrz, to zapewniam sobie ten dostęp sam. Zawsze za pomocą rozwiązań, które uważam za godne zaufania.